Installing Awesome 4.0 on Ubuntu 16.04

Output of `awesome --version`: Haven't installed it yet.  Trying to install 4.0.

I'm a noob to Awesome, so what am I doing wrong?  I'm 90% sure I installed all of the deps.  That is, I've installed everything where it says "checking for modules"

I run `make` and this happens:
```
Running cmake…
-- git -> /usr/bin/git
-- asciidoc -> /usr/bin/asciidoc
-- xmlto -> /usr/bin/xmlto
-- gzip -> /bin/gzip
-- ldoc -> /usr/bin/ldoc
-- convert -> /usr/bin/convert
-- Checking for modules 'glib-2.0;gdk-pixbuf-2.0;cairo;x11;xcb-cursor;xcb-randr;xcb-xtest;xcb-xinerama;xcb-shape;xcb-util>=0.3.8;xcb-keysyms>=0.3.4;xcb-icccm>=0.3.8;xcb-xkb;xkbcommon;xkbcommon-x11;cairo-xcb;libstartup-notification-1.0>=0.10;xproto>=7.0.15;libxdg-basedir>=1.0.0;xcb-xrm'
--   
CMake Error at /usr/share/cmake-3.5/Modules/FindPkgConfig.cmake:367 (message):
  A required package was not found
Call Stack (most recent call first):
  /usr/share/cmake-3.5/Modules/FindPkgConfig.cmake:532 (_pkg_check_modules_internal)
  awesomeConfig.cmake:153 (pkg_check_modules)
  CMakeLists.txt:17 (include)


-- Configuring incomplete, errors occurred!
See also "/home/legostax/awesome-4.0/.build-LSUbGn-x86_64-linux-gnu-5.4.0/CMakeFiles/CMakeOutput.log".
See also "/home/legostax/awesome-4.0/.build-LSUbGn-x86_64-linux-gnu-5.4.0/CMakeFiles/CMakeError.log".
Makefile:22: recipe for target 'cmake' failed
make: *** [cmake] Error 1
```

Please forgive me if this is the wrong place to be asking for help.

--- a/awesomeConfig.cmake
+++ b/awesomeConfig.cmake
@@ -142,6 +142,7 @@ set(AWESOME_DEPENDENCIES
 
 # Check the deps one by one
 foreach(dependency ${AWESOME_DEPENDENCIES})
+    unset(TMP_DEP_FOUND CACHE)
     pkg_check_modules(TMP_DEP REQUIRED ${dependency})
 
     if(NOT TMP_DEP_FOUND)

